packages/core/src/app/payment/hostedCreditCard/withHostedCreditCardFieldset.tsx

```diff
@@ -13,7 +13,7 @@ import { CheckoutContextProps, PaymentFormValues } from '@bigcommerce/checkout/p
 
 import { withCheckout } from '../../checkout';
 import { connectFormik, ConnectFormikProps } from '../../common/form';
-import { MapToPropsFactory } from '../../common/hoc';
+import { MapToPropsFactory } from '@bigcommerce/checkout/legacy-hoc';
```

Comment:
Hi @richapatel1510 , if you got some time, could you please check why our linter did not pick up errros like this? Something seems missing.
Ideally, importing interal pakcages should be placed above this group.

Reply:
For example, https://github.com/bigcommerce/checkout-js/blob/master/packages/core/src/app/checkout/Checkout.tsx

Reply:
@bc-peng Fixed the import order for current changes. And it did not pick import order errors because they are labelled as warning and not error in `https://github.com/bigcommerce/eslint-config/blob/master/packages/eslint-config/configs/base.js`
![Screenshot 2023-08-21 at 5 33 14 pm](https://github.com/bigcommerce/checkout-js/assets/141614330/331281fc-14f6-4d1b-ae74-1bf00ace9962)

Reply:
We can change the config in another PR.
Thanks for your investigation.